src/DIPS.Xamarin.UI/Controls/Toast/ToastOptions.cs

```diff
@@ -0,0 +1,41 @@
+using System;
+using System.Threading.Tasks;
+using DIPS.Xamarin.UI.Internal.Xaml;
+using Xamarin.Forms;
+
+namespace DIPS.Xamarin.UI.Controls.Toast
+{
+    /// <summary>
+    ///     Options to setup in the Toast control
+    /// </summary>
+    public class ToastOptions : BindableObject
+    {
+        /// <summary>
+        ///     Performs action on tapping the toast
+        ///     <remarks>Will Override closing the toast on tapping. Default action is to close the Toast</remarks>
+        /// </summary>
+        public Action ToastAction { get; set; } = async () => await Toast.Current.CloseToastAsync();
+
+        /// <summary>
+        ///     Animation on displaying the Toast
+        ///     <remarks>Default animation is Fading-In in 250 ms</remarks>
+        /// </summary>
+        public Func<ToastView, Task> DisplayAnimation { get; set; } = toast =>
+        {
+            toast.Opacity = 0;
+            return toast.FadeTo(1, 500, Easing.Linear);
+        };
+
+        /// <summary>
+        ///     Animation on closing the Toast
+        ///     <remarks>Default animation is Fading-Out in 250 ms</remarks>
+        /// </summary>
+        public Func<ToastView, Task> CloseAnimation { get; set; } = toast => toast.FadeTo(0, 500, Easing.Linear);
+
+        /// <summary>
+        ///     Hide the toast automatically after the given milliseconds
+        ///     <remarks>If value is 0, toast won't be hidden automatically. Default value is 3000 ms</remarks>
+        /// </summary>
+        public int HideToastIn { get; set; } = 3000;
```

Comment:
Yes 👍

Reply:
I was a little surprised that  `0` meant that it would never go away. `0` in my head felt like no time, meaning it would just display and the hide. Maybe we a `-1` (or any negative value) would be a better number to make it never hide?

Reply:
Since `0` doesn't make sense, picked the value `0`. It's documented.

Reply:
I don't agree. I think `0` makes sense. `Duration=0` means it won't have any duration (meaning it will show and hide immediately). What doesn't make sense it for it to never disappear in my opinion.